EN: Big load of stuff from the update

DabirSA opened the pull request.

DabirSA committed:
fix line breaks

DabirSA committed:
wrong kind of customization

DabirSA committed:
reduce line count on some arks missions

DabirSA committed:
correct which fang to kill

DabirSA committed:
Merge remote-tracking branch 'upstream/EN' into EN

DabirSA committed:
episode change requirements

DabirSA committed:
stratos quest NPC autowords

DabirSA committed:
episode stratos tweaks and fixes

DabirSA committed:
episode alma tweaks and fixes

DabirSA committed:
Fang Banshee, not Rockbear

DabirSA committed:
mission promotion

DabirSA committed:
remove extra empty line

DabirSA committed:
feed aspell

DabirSA committed:
Matter Board tutorial

DabirSA committed:
move to right folder

DabirSA committed:
tweaks to ep5

DabirSA committed:
PA and tech updates

DabirSA committed:
fix bizarre capitalisation change

DabirSA committed:
spelling

DabirSA committed:
Fix more messed up capitalisation

DabirSA committed:
deband affects DEF

DabirSA committed:
Update other copy of level 80 cap tutorial

DabirSA committed:
shield defects

DabirSA committed:
ep5 mid mission dialogue

DabirSA committed:
dialogue tweaks

DabirSA committed:
missing quest descriptions

DabirSA committed:
other copy of quest descriptions and failure conditions

DabirSA committed:
move this

DabirSA committed:
end of ep5 tweaks and mid-mission dialogue

DabirSA committed:
ep 6 checks

DabirSA committed:
fix typo

DabirSA committed:
change labels for phantom skill params

DabirSA committed:
Merge remote-tracking branch 'upstream/EN' into EN

SynthSy commented on Quests/Story EP6 names/Chapter 1/quest_00301440.csv:
```diff
@@ -1,4 +1,4 @@
-Title#0,"""Assault of the «Phantom»"""
+Title#0,"""Assault of the \""Phantom\"""""
```
Revert that, it was stylized like that on purpose.

DabirSA replied:
Reverted.

DabirSA committed:
revert as requested

DabirSA committed:
Merge remote-tracking branch 'upstream/EN' into EN

DabirSA committed:
Rename apc_chat folder for organisation and clarity

DabirSA committed:
move astarte dialogue

DabirSA committed:
use henri to find out what pictograms look like

DabirSA committed:
Merci.

DabirSA committed:
Astarte mission instructions

DabirSA committed:
clarify stealth tech charge

Wasn't clear which bits of the effect are temporary and which last as long as you continue to charge the tech.

DabirSA committed:
fix section numbers

DabirSA committed:
this line changed

DabirSA committed:
check matoi dialogue

DabirSA committed:
Check Melrondia's dialogue

DabirSA committed:
fix quote

DabirSA committed:
fix capitalisation

DabirSA committed:
more spelling

DabirSA committed:
missing space

DabirSA committed:
Kyokuya greetings

DabirSA committed:
Update MBD tutorial

DabirSA committed:
Other copy of enemy trigger tutorial

DabirSA committed:
Update other copy of challenge block tutorial

DabirSA committed:
Update item-related tutorials

DabirSA committed:
UQ tutorial

DabirSA committed:
More Rockbears

DabirSA committed:
first ep4 story quest description

DabirSA committed:
Update config.csv

DabirSA committed:
more astarte arks mission intros

DabirSA committed:
second copy of UH free field tutorial

DabirSA committed:
phantom tutorial

DabirSA committed:
CO notice for kyokuya

DabirSA committed:
new ring skills

DabirSA committed:
Merge remote-tracking branch 'upstream/EN' into EN

DabirSA committed:
fix spelling

DabirSA committed:
clean up

DabirSA committed:
fix line breaks

DabirSA committed:
more cleanup on phaleg

DabirSA committed:
ep3 afin partner notice

DabirSA committed:
Update quest_00010280.csv

DabirSA committed:
phantom tutes

DabirSA committed:
Merge remote-tracking branch 'upstream/EN' into EN

DabirSA committed:
fix extra line break

DabirSA committed:
trigger item name change

DabirSA committed:
fix punctuation clipping

DabirSA committed:
title renamed

DabirSA committed:
new hero CO reqs

DabirSA committed:
kyokuya lobby dialogue

DabirSA committed:
clipping

DabirSA committed:
tweak talis short charge description

DabirSA committed:
Translate tutorial popups

DabirSA committed:
Translate new Koffie messages

DabirSA committed:
tweak Necky quest instructions

DabirSA committed:
tweak necky quest dialogue

DabirSA committed:
ep1 tweaks and fixes

DabirSA committed:
Put omnibus files back in their episodes

DabirSA committed:
Merge remote-tracking branch 'upstream/EN' into EN

DabirSA committed:
completely misunderstood shift kugelsturm at some point

DabirSA committed:
Update COs

DabirSA committed:
Fix

DabirSA committed:
Astarte's support partner tutorial

DabirSA committed:
translate xiera nagging about training

DabirSA committed:
some character creator stuff

DabirSA committed:
couple of tip tweaks

DabirSA committed:
Merge remote-tracking branch 'upstream/EN' into EN

DabirSA committed:
sturm mizer death tips

DabirSA committed:
Dral Veluge death tips

DabirSA committed:
Rideroid 14* partizan potential name

DabirSA committed:
Rafaga Ringahda death tips

DabirSA committed:
grana gwanahda death tips

DabirSA committed:
nails -> claws

DabirSA committed:
not granada

DabirSA committed:
gwanahda

DabirSA committed:
Merge remote-tracking branch 'upstream/EN' into EN
